rpm/brave-keyring.spec

```diff
@@ -24,27 +22,25 @@ fetching the packages.
 
 %install
 mkdir -p %{buildroot}/etc/pki/rpm-gpg
-mkdir -p %{buildroot}/etc/cron.daily
+mkdir -p %{buildroot}/tmp
 mkdir -p %{buildroot}/usr/lib/sysctl.d/
 install -m 644 etc/pki/rpm-gpg/RPM-GPG-KEY-brave -t %{buildroot}/etc/pki/rpm-gpg/
 install -m 644 etc/pki/rpm-gpg/RPM-GPG-KEY-brave-beta -t %{buildroot}/etc/pki/rpm-gpg/
 install -m 644 etc/pki/rpm-gpg/RPM-GPG-KEY-brave-nightly -t %{buildroot}/etc/pki/rpm-gpg/
-install -m 755 etc/cron.daily/brave-key-updater -t %{buildroot}/etc/cron.daily/
+install -m 755 tmp/brave-key-updater -t %{buildroot}/tmp/
```

Comment:
This should be considered as a one-off script and from that standpoint I don't think we should install the script. Also, I don't like the idea of `cron` picking it up and trying to run it daily or even storing the script in a `cron` related directory when `cron` might not even be installed in the target OS.

Do you see an issue using `/tmp` to store a temporary script?

Reply:
rpm has a post-install script hook you can use to set things up, although I suspect you're not supposed to make network requests there and it has to be written in posix sh script.

Otherwise, I agree with @wknapik that any utility scripts should be installed somewhere permanent (maybe [/usr/libexec](https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s07.html)?) even if we only run them once.

Reply:
Yes, after checking some documentation I see clearly why deleting the file outside RPM would not be correct. Why not install it in `%{_bindir_}/brave-key-updater` then? It would run once triggered by the post-install script and then stay there. It makes more sense to me to install it among the system binaries than in a cron location.

Reply:
Sounds reasonable!

Reply:
I'd like to avoid the script to show as an autocomplete option for the `brave` prefix but I think `/usr/lib` and `/usr/share` wouldn't be the correct places either since the former is for libraries and the latest for non-binaries.

Reply:
What about `/usr/libexec`? 

>The /usr/libexec directory is used for storing system executables that are not intended to be invoked directly by users or shell scripts.